.github/workflows/container.yml

```diff
@@ -0,0 +1,32 @@
+name: Container
+
+on:
+  - pull_request
+  - push
+
+jobs:
+  build:
+    name: "📦 Build container"
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Build the container
+        run: docker build -t osbuild-installer -f distribution/Dockerfile-ubi .
+
+      - name: Run the container
+        run: |
+          docker run -d --net=host --name osbuild-installer osbuild-installer
+          docker ps
+
+      - name: Verify that the startup log line is in the logs
+        run: |
+          docker logs osbuild-installer | grep "Starting osbuild-installer server"
+
+      - name: Check that the API is responding
+        run: curl -sv http://127.0.0.1:8086/api/osbuild-installer/v1/version
+
+      - name: Stop the container
+        run: docker stop --time 1 osbuild-installer
```

Comment:
Could you add a single newline here at the end of the file?

Reply:
Done!